apps/modernization-ui/src/validation/entry/validPageNameRule.ts

```diff
@@ -0,0 +1,14 @@
+import { maxLengthRule } from './maxLengthRule';
+
+/**
+ * Creates a standardized react-hook-form rule for the entry of name values.
+ */
+const validPageNameRule = {
+    ...maxLengthRule(),
+    pattern: {
+        value: /^[\w*()_+\-=;:/.,\d\s]+$/gm,
```

Comment:
Suggestion: i think this can be simplified slightly.
`\d` is equivalent to [0-9]
`\w` is equal to:  letters (A–Z, a–z), numbers (0–9), and underscore (_). [source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape#w).

So using both `\w` and `\d` is redundant. And `\w` includes underscore.

With that we could update the regex to be: `\w*()+\-=;:/.,\s`